WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MePage.java

```diff
@@ -55,7 +55,7 @@ public MePage verifyUsername(String username) {
     }
 
     public boolean isSelfHosted() {
-        waitForElementToBeDisplayed(R.id.row_logout);
+        waitForElementToBeDisplayed(R.id.row_app_settings);
```

Comment:
Updated to wait for a menu item that's listed near the top of the menu instead of the last item.

Reply:
👋 @jostnes !

I understand that this fixes UI tests as the `Logout` button is not displayed when running on Pixel 2 (the device used on Firebase), which caused failures when trying to log out. But why is that happening now, and becomes a requirement for this PR, I guess this is just `Jetpack` UI test related, right? 🤔 

Also, as part of this change you removed 3 of `waitForElementToBeDisplayed(...)`, 2 from `WPSupportUtils.scrollToThenClickOn(...)` and 1 from `MePage.logout()`. Why was that a requirement for this PR? 🤔